docs/windows.md

```diff
@@ -0,0 +1,197 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay network with VXLAN tunneling to
```

Comment:
Did not get the answer for this one: we support only VXLAN or also other tunnel types?

Reply:
@jianjuns sorry for the delay.  We have not test other tunnel types before.

Today I check the OVS doc and do the verification, I think we can support VXLAN and STT tunnels on windows.